redis/src/main/scala/zio/redis/options/Streams.scala

```diff
@@ -59,6 +59,8 @@ trait Streams {
 
   sealed case class StreamEntry[I, K, V](id: I, fields: Map[K, V])
 
+  sealed case class Stream[N, I, K, V](name: N, entries: Chunk[StreamEntry[I, K, V]])
```

Comment:
This class name is arguable. I accept proposals for better))
It has conflict wit zio.stream.Stream, I have to fix imports((

Reply:
StreamChunk now